### DISCUSSION TITLE
Range Result Selection

### DISCUSSION THREAD

**Discussion by raindropsfromsky** [Flow-Launcher/Flow.Launcher#1169] [Ideas]
Often the user needs to select multiple files from the search results and copy/move them to another folder.
Is it possible to select multiple files from the search result and press CTRL+C or CTRL+X to put all selected files on the clipboard?

**Comment by taooceros** [comment]
It requires a significant change of the UI/UX and the backend code for that. I wish to have that, but probably not easy to be done for now.

**Reply by jjw24** [reply]
Just a note this may be achievable with adding QuickLook preview functionality (via plugin) which if I remember correctly allows you to select folders and do directory operations on them.
